docs/k0sctl-install.md

```diff
@@ -0,0 +1,171 @@
+
+## Deploying a k0s cluster using k0sctl
+
+k0sctl is a command-line tool for bootstrapping and the management of k0s clusters. Installation instructions can be found in the [k0sctl github repository](https://github.com/k0sproject/k0sctl#installation).
+
+k0sctl will connect to provided host using ssh and gather information about the host. Based on the finding it will proceed to configure the host in question and install k0s binary.  
```

Comment:
```suggestion
k0sctl will connect to hosts listed in a configuration file via ssh and gather information about them. Based on the findings it will then proceed to configure the hosts to create a k0s cluster.  
```

Reply:
> Suggested change 

> k0sctl will connect to hosts listed in a configuration file via ssh and gather information about them. Based on the findings it will then proceed to configure the hosts to create a k0s cluster.

I like the phrasing suggested by @kke.